pages/guides/_no-search-results.html

```diff
@@ -1,12 +1,8 @@
-<div id="no-matches"  class="guides-section-white">
-    <div class="row text-center">
-        <div class="no-results no-results-space">
-            <div class="col-md-6 col-md-offset-3" >
-                <img src="/assets/img/guides/layout/no-search-results.png" alt="no search results" /><br />
-                <label>No Results Found</label>
-                <p>We couldn’t find any guides matching your search. Contact us and we can help you find the answers you’re looking for.</p>
-                <a href="/contact"><button type="button" class="btn btn-primary">Contact Us</button></a>
-            </div>
-        </div>
+<div id="no-matches" class="no-results no-results-space center">
```

Comment:
This change doesn't seem related to this PR

Reply:
No it's not. I noticed it wasn't well-positioned in the middle while I was working on this and decided to fix it at once. I can take it out and create a separate PR.